rules/REQUEST-941-APPLICATION-ATTACK-XSS.conf

```diff
@@ -838,8 +838,8 @@ SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES_NAMES|ARGS_NAME
     msg:'Node-Validator Deny List Keywords',\
     logdata:'Matched Data: %{TX.0} found within %{MATCHED_VAR_NAME}: %{MATCHED_VAR}',\
     tag:'application-multi',\
-    tag:'language-multi',\
     tag:'platform-multi',\
+    tag:'language-multi',\
```

Comment:
Why?

Reply:
Idk. I didn't change it. I just did 'regex update' and push.

Reply:
That's weird... Nothing but the regex should be touched. Please revert this change.

Reply:
Please revert the change in tags.

Reply:
I already did. For some reason it's not showing here. Try merging.